### PR TITLE
Add scroll to top button

### PR DESCRIPTION
**Pull request by ankheur**
Adding a scroll to top button on the homepage. Styling can surely be improved though.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,35 +237,35 @@ <h3 class="center">Contributors</h3>
         <ul class="normalize-ul center">
           <li>
             <img class="profile-picture" src="https://github.com/matthiasliszt.png" alt="Matthias Liszt" />
-            <a href="https://matthiasliszt.github.io">Matthias Liszt</a>
+            <a href="https://matthiasliszt.github.io" target="_blank" rel="noopener">Matthias Liszt</a>
```
Comment (robaxelsen) [inline index.html]: Could you please also add noreferrer; `rel="noopener noreferrer"`?

**Review by robaxelsen — CHANGES_REQUESTED** [review]
Great PR, thanks a lot! See my review comment, other than that LGTM :+1:

File: index.html (hunk 1 above)
Comment (ankheur) [reply index.html]: I actually realised after creating the commit that no other links on the page has the target=_blank attribute and thought that there might be a reason so I can remove these ones instead to have the same behaviour on all links (or add the target=_blank on all links, either way)

Comment (robaxelsen) [reply index.html]: Good observation, but let's keep it with target="_blank". We can always change the other external links to same behavior.

Comment (robaxelsen) [reply index.html]: Disregard my previous comment. I misunderstood. Leave as is.

**Review by robaxelsen — APPROVED** [review]

**Comment by robaxelsen** [comment]
Thanks a lot for the contribution :heart:

**Pull request merged**